backend/migrations/versions/7f254b9daa12_create_schooldistrict_model.py

```diff
@@ -0,0 +1,36 @@
+"""Create SchoolDistrict model
+
+Revision ID: 7f254b9daa12
+Revises: f92a559349ff
+Create Date: 2024-02-12 13:08:38.325282
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '7f254b9daa12'
+down_revision = 'f92a559349ff'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('school_district',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('active', sa.Boolean(), nullable=False),
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.Column('updated_at', sa.DateTime(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('name')
```

Comment:
Just as FYI -- I confirmed experimentally that different casings will not be treated as violations as we discussed recently.